godtools/Localizable.strings

```diff
@@ -0,0 +1,24 @@
+/* 
+  Localizable.strings
```

Comment:
I found a snippet for the extension, and about the methods on the label and buttons classes that is something I figured out. Do you feel comfortable with it?